ansible/plugins/connection/multi_passwd_ssh.py

```diff
@@ -63,7 +64,11 @@ def _conn_with_multi_pwd(self, *args, **kwargs):
             self.set_option("password", conn_password)
             self._play_context.password = conn_password
             try:
-                return func(self, *args, **kwargs)
+                results = func(self, *args, **kwargs)
+                if not self.has_option("current_password_hash"):
```

Comment:
Seems like there was issue here:
`An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Connection' object has no attribute 'has_option'`
https://elastictest.org/scheduler/testplan/66052cd9500f341a8721504a?testcase=testbed_q_sonic-elastictest-prod-vmss-D8s-v3_82539_vms-kvm-t0_prepare.log&type=prepare

Reply:
Fixed. The reason is that I tested the change using ansible v2.13.13. The ansible version used by PR checker is still v2.8.20 which does not have `self.has_options` yet.